Gruntfile.js

```diff
@@ -147,18 +244,23 @@ module.exports = function (grunt) {
         },
 
         clean: {
-            doc: ['doc']
+            doc: ['doc'],
+            visualTests: ['visual-tests/dist']
         }
     });
 
     require('matchdep').filterDev('grunt-*').forEach(grunt.loadNpmTasks);
+    grunt.loadNpmTasks('assemble');
```

Comment:
Yes, as we're actually using _assemble_, rather than _grunt-assemble_:

https://www.npmjs.com/package/assemble

Might be worth using [load-grunt-tasks](https://github.com/sindresorhus/load-grunt-tasks) to make this a bit tidier?